lib/kafka_ex/server.ex

```diff
@@ -313,9 +318,13 @@ defmodule KafkaEx.Server do
     end
   end
 
+  defp offset_commit(state = %State{consumer_group: false}, offset_commit_request) do
```

Comment:
all the `consumer_group: false` will be updated to `consumer_group: :no_consumer_group`, also this is a hack, I am beginning to think `KafkaEx.offset_fetch` and `KafkaEx.offset_commit` should not be an exposed API